<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorValue</class>
 <widget class="QWidget" name="ColorValue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>142</width>
    <height>74</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QLabel" name="colorRGB">
     <property name="text">
      <string>R:255 G:255 B:255</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QLabel" name="colorHex">
     <property name="text">
      <string>#FFFFFF</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="colorPre">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
